spec/models/board_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
Removes these lines:
```
it { should belong_to(:project).touch(true) }
it { should validate_presence_of(:project) }
```

Reply:
can you please add a reminder Task to ensure that after ce-sync, we don't loose these two in Pro?